### DISCUSSION TITLE
关于工具栏配置，修改现有的toolbar的title

### DISCUSSION THREAD

**Discussion by haoshifu** [wangeditor-next/wangEditor-next#378] [Q&A] [answered]
## 问题描述
关于工具栏配置，修改现有的toolbar的title

## wangEditor 版本

wangEditor-next

## 是否查阅了文档 ？

（文档链接 [https://cycleccc.github.io/docs/](https://cycleccc.github.io/docs/) ）

是

## 最小成本的复现步骤

想要将默认行高的显示修改为行高，目前定位不到title的修改位置，请问作者，这个具体的修改应该在哪进行？
<img width="344" alt="行高修改" src="https://github.com/user-attachments/assets/8deef0fd-9001-4e02-b2f1-545e9da8b4af">

- 步骤一
- 步骤二
- 步骤三

**Comment by cycleccc** [comment, marked as answer]
通过 i18n 修改即可
https://github.com/wangeditor-team/wangEditor/issues/4555#issuecomment-1188614814

**Comment by cycleccc** [comment]
工具栏 title 可通过 i18n 修改，已有对应说明。本讨论现关闭。
